packages/react-components/src/InputSection.tsx

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017-2020 @polkadot/react-components authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+
+interface InputSectionProps {
+  children: ReactNode,
+  className?: string,
+}
+
+const InputSection = ({ children, className }: InputSectionProps) => (
```

Comment:
Always use a function, no `const` shortcuts. (consistency & mainatainability, i.e. there is just one way things look)